projects/igniteui-angular/src/lib/grids/cell.component.ts

```diff
@@ -722,6 +823,48 @@ export class IgxGridCellComponent implements OnInit, OnChanges, OnDestroy, CellT
                 cssProps: {} /* don't disable user-select, etc */
             } as HammerOptions);
         }
+
+    }
+
+    public ngAfterViewInit() {
+        this.errorTooltip.changes.subscribe(() => {
+            if (this.errorTooltip.length > 0 && this.active) {
+                // error ocurred
+                this.cdr.detectChanges();
+                this.openErrorTooltip();
+            }
+            this.grid.validationStatusChange.emit(
```

Comment:
@MayaKirova I'm a bit confused - this is a grid level event, right? Yet it emits for every cell edit change?

Reply:
It's because we agreed that it should trigger the moment an invalid value is entered in an editor. Here:
https://github.com/IgniteUI/igniteui-angular/issues/6501#issuecomment-1177878614